### DISCUSSION TITLE
Question about security analyst install

### DISCUSSION THREAD

**Discussion by Sgtkeebs** [Security-Onion-Solutions/securityonion#7833] [Unsupported Versions] [answered]
Hello everyone, 

I am a student, I am currently taking the Blue Team level 1 exam and I am also signed up with Blue Team Labs Online which is a TryHackMe like environment where I can practice investigations, ctf's. I would like to install the Security Analyst version of Security Onion. I do not need an IDS or NIDS, I just need to evaluate pcap files, syslog logs, among other things. I am confused about the Security Onion analyst setup. Do I need to setup an Ubuntu vm and then install the analyst version or do I need to use the Security Onion ISO to install the analyst version? Sorry if this is a dumb question, I am new to all of this

**Comment by dougburks** [comment, marked as answer]
Download our Security Onion ISO image and then follow the screenshots at https://docs.securityonion.net/en/2.3/first-time-users.html.

(The screenshots are for our upcoming 2.3.120 release but it should still be the same process.)
